conans/test/utils/tools.py

```diff
@@ -837,7 +837,7 @@ def run(self, command_line, user_io=None, assert_error=False):
             added_modules = set(sys.modules).difference(old_modules)
             for added in added_modules:
                 sys.modules.pop(added, None)
-        self._handle_cli_result(command, assert_error=assert_error, error=error)
+        self._handle_cli_result(command_line, assert_error=assert_error, error=error)
```

Comment:
This error was introduced here: https://github.com/conan-io/conan/pull/5927/files#diff-d59b3bae8dcde0d1ebfd1de3dc21d92fR840